packages/cli/src/commands/governance/propose.ts

```diff
@@ -16,7 +17,21 @@ export default class Propose extends BaseCommand {
 
   static flags = {
     ...BaseCommand.flags,
-    jsonTransactions: flags.string({ required: true, description: 'Path to json transactions' }),
+    jsonTransactions: flags.string({
+      exclusive: ['interactive'],
+      description: 'Path to json transactions',
+    }),
+    interactive: flags.boolean({
+      default: false,
+      exclusive: ['jsonTransactions'],
+      description:
+        'Form proposal using an interactive prompt for Celo registry contracts and functions',
```

Comment:
looks like the descriptions are swapped

Reply:
Hmm, the exclusive property is to specify that the flags are mutually exclusive but I think the descriptions are paired correctly

Reply:
oh oops i see, missread